packages/client/foreman_ygg_migration/foreman_ygg_migration.spec

```diff
@@ -0,0 +1,34 @@
+Name: foreman_ygg_migration
+Version: 0.0.1
+Summary: A helper package to ease transition from yggdrasil 0.2.z to 0.4.z
+Release: 1%{?dist}
+License: MIT
+
+%if 0%{?rhel} >= 8
+Supplements: yggdrasil >= 0.4.0
+%endif
+Conflicts:   yggdrasil  < 0.4.0
+
+%description
+A helper package to ease transition from yggdrasil 0.2.z to 0.4.z
+
+%post
+if systemctl is-enabled yggdrasild >/dev/null 2>/dev/null; then
+	grep -Pq '^server'    /etc/yggdrasil/config.toml || sed -i 's/broker.*=/server =/' /etc/yggdrasil/config.toml
+	grep -Pq '^prefix'    /etc/yggdrasil/config.toml || echo 'prefix = "yggdrasil"' >>/etc/yggdrasil/config.toml
+	grep -Pq '^data-host' /etc/yggdrasil/config.toml || echo 'data-host = ""' >>/etc/yggdrasil/config.toml
+
+	if grep -q FOREMAN_YGG_WORKER_WORKDIR /etc/systemd/system/yggdrasild.service.d/* 2>/dev/null; then
+		mkdir -p /etc/systemd/system/com.redhat.Yggdrasil1.Worker1.foreman.service.d/
+		cp -r /etc/systemd/system/yggdrasild.service.d/* /etc/systemd/system/com.redhat.Yggdrasil1.Worker1.foreman.service.d/
+	fi
+
+	systemctl disable --now yggdrasild
```

Comment:
Shouldn't the old service get uninstalled and thus removed?

Reply:
From what I've seen, we're missing systemd scriptlets in our yggdrasil spec, so while the old package gets uninstalled and the `yggdrasild` service definition goes away, it seems to stay enabled.